### PR TITLE
zgrep: add page

### PR DESCRIPTION
**Pull request by RH-sdavey**
https://github.com/tldr-pages/tldr/issues/3966

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/zgrep.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# zgrep
+
+> Grep pattern(s) of text (strings/regex) from files within compressed file (equivalent to grep -Z).
+> Syntax: zgrep [grep_options] pattern file.
```
Comment (zdroid) [inline pages/linux/zgrep.md:4]:
```suggestion
```
This is redundant, as you can see it in the examples. We omit that in tldr pages.

File: pages/linux/zgrep.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# zgrep
+
+> Grep pattern(s) of text (strings/regex) from files within compressed file (equivalent to grep -Z).
```
Comment (zdroid) [inline pages/linux/zgrep.md:3]:
```suggestion
> Grep text patterns from files within compressed file (equivalent to grep -Z).
```

File: pages/linux/zgrep.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> Syntax: zgrep [grep_options] pattern file.
+
+- Grep pattern from compressed file (case-sensitive):
```
Comment (zdroid) [inline pages/linux/zgrep.md:6]:
```suggestion
- Grep a pattern in a compressed file (case-sensitive):
```
Should be similar to this in other examples, too.

File: pages/linux/zgrep.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`zgrep pattern path/to/compressed/file`
```
Comment (zdroid) [inline pages/linux/zgrep.md:8]:
```suggestion
`zgrep {{pattern}} {{path/to/compressed_file}}`
```
Missing braces for variable arguments. Should be similar in other examples too.

File: pages/linux/zgrep.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Grep pattern from compressed file (case-insensitive):
+
+`zgrep -i pattern path/to/compressed/file`
```
Comment (einverne) [inline pages/linux/zgrep.md:12]: need braces here

File: pages/linux/zgrep.md (hunk 1 above)
Comment (RH-sdavey) [reply pages/linux/zgrep.md:4]: removed

File: pages/linux/zgrep.md (hunk 3 above)
Comment (RH-sdavey) [reply pages/linux/zgrep.md:6]: changed

File: pages/linux/zgrep.md (hunk 2 above)
Comment (RH-sdavey) [reply pages/linux/zgrep.md:3]: changed

File: pages/linux/zgrep.md (hunk 4 above)
Comment (RH-sdavey) [reply pages/linux/zgrep.md:8]: changed, only line I wasnt sure about was line 23, I asked in gitter/tldr room. But used my best judgement, please check line 23

**Comment by zdroid** [comment]
I had to do just one fix, it's all nice!

**Comment by RH-sdavey** [comment]
Yes, sorry I missed that, thanks for catching it @zdroid 
Ill be back for another contribution asap. Thanks for help and great community spirit!

**Pull request merged**